.travis.yml

```diff
@@ -1,33 +0,0 @@
-dist: trusty
-
-jdk:
-  - openjdk11
-
-language: android
-android:
-  components:
-    - build-tools-30.0.2
-    - android-30
```

Comment:
Don't need to set this either?

Does this figure out the versions out automatically somehow based on gradle config?

Reply:
This was to tell Travis what Android version to install on the build image. I think all versions of the Android SDK are installed on the GitHub build image.

This seems to be what determines which Android version the build uses: https://github.com/seedvault-app/seedvault/blob/master/app/build.gradle#L20

Reply:
Sweet